src/pages/TrueSightV2/components/SimpleTooltip.tsx

```diff
@@ -185,7 +187,9 @@ export default function SimpleTooltip({
       document.body,
     )
   }, [alphaLeft, className, disappearOnHover, handleMouseLeave, left, maxWidthProp, text, top, widthProp])
-
+  if (disabled) {
+    return <>{children}</>

```

Comment:
why not `return children`

Reply:
in case children = undefined, it will throw an error. Wrapped with fragment is a good practice